### ISSUE TITLE
Reword Data Preparation Quiz

### ISSUE THREAD

**Issue by dsilver829** [udacity/sdc-issue-reports#1302]
The Data Preparation Quiz in the Object Detection lesson matches "Splitting the data into a training and test set" with "To avoid overfitting / improve generalization". I don't think that's quite right. It's more like, "To provide an estimate of how well the model will handle new data"

**Comment by baumanab** [comment]
The option has now been changed to:

> To estimate generalization of the model to new data
